modules/Hoot/ui/managePanel/jobs.js

```diff
@@ -364,10 +377,10 @@ export default class Jobs extends Tab {
 
     populateJobsHistory( jobs ) {
```

Comment:
When there are jobs present they aren't shown.  There is a line of code further down that needs to be adjusted now that we are passing the jobs object and not the jobs array.